integration_test/testutil_test.go

```diff
@@ -37,8 +37,10 @@ import (
 const (
 	persistentIndexCache = "krew-persistent-index-cache"
 	krewBinaryEnv        = "KREW_BINARY"
-	validPlugin          = "konfig" // a plugin in central index with small size
-	validPlugin2         = "mtail"  // a plugin in central index with small size
+	validPlugin          = "konfig"          // a plugin in central index with small size
+	validPlugin2         = "mtail"           // a plugin in central index with small size
+	LocalhostURL         = "127.0.0.1:60564" // make sure this port is open. incase of editing port number, make sure to update the URL in testdata/konfig_localhost.yaml
```

Comment:
you shouldn't need this. just simplify by reading tarball from `--archive` in integration test?

Reply:
As we discussed previously `--archive` cannot be used with `--manifest-url`. Hence, I added this.